tensorflow_datasets/core/features/feature.py

```diff
@@ -724,16 +724,25 @@ def get_inner_feature_repr(feature):
   else:
     return repr(feature)
 
+def _repr_html_batch(ex) -> str:
+  """Default HTML batch repr."""
+  batch_ex = ''
+  for value in ex:
+    batch_ex += _repr_html(value) + ', '
+  return ("<div style='overflow-x: scroll;"
+          "white-space: nowrap; width: 90px;"
+          "padding:2em 0em 2em 0em; float: right'>"
+          f"{type(ex).__qualname__}([{batch_ex[:-2]}])" # exclude last ,[space]
```

Comment:
The `type(ex).__qualname__` is already added in the `_repr_html(value)` above, so it should not be re-added here. Simply concatenating the string is good.